packages/frontend/src/components/ProjectConnection/WarehouseSettingsForm.tsx

```diff
@@ -42,7 +42,7 @@ const WarehouseSettingsForm: FC<WarehouseSettingsFormProps> = ({
 }) => {
     const warehouseType: WarehouseTypes = useWatch({
         name: 'warehouse.type',
-        defaultValue: WarehouseTypes.BIGQUERY,
+        defaultValue: WarehouseTypes.POSTGRES,
```

Comment:
We should keep Bigquery as the default since it's the majority of the users use it.

Reply:
I think you forgot to update this value.